connect-button/src/main/java/com/ifttt/connect/ui/StartIconDrawable.java

```diff
@@ -55,9 +54,6 @@
         this.iconSize = iconSize;
         this.initialBackgroundSize = initialBackgroundSize;
         this.onDarkBackground = onDarkBackground;
-        this.startIconBackgroundColor =
-                onDarkBackground ? ContextCompat.getColor(context, R.color.ifttt_start_icon_background_on_dark)
-                        : Color.BLACK;
```

Comment:
is this intentional? This color is used below, if we don't set it in the constructor, how can we use it where required?

Reply:
We no longer use this field because we are going to use the `backgroundColor`, which will be set later on in the process, to derive the background color for the start icon.